common/protob/messages-common.proto

```diff
@@ -62,11 +62,12 @@ message ButtonRequest {
         ButtonRequest_PublicKey = 11;
         ButtonRequest_MnemonicWordCount = 12;
         ButtonRequest_MnemonicInput = 13;
-        ButtonRequest_PassphraseType = 14;
+//        ButtonRequest_PassphraseType = 14;  DEPRECATED
```

Comment:
Is this the right way or do we just remove it completely?

Reply:
i'd keep it like this

this whole thing will go away soon anyway .)